### PR TITLE
Update the Cryptocurrency Demo

### PR DESCRIPTION
**Pull request by ufukyilmaz**
- Add the new Twitter Connector to the cryptocurrency demo.

### CODE REVIEW

File: cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java [hunk 1]
```diff
@@ -45,38 +51,38 @@
...
- *    ┌────────────────────────┐   ┌────────────────────────┐
- *    │    Calculate 5min      │   │    Calculate 30sec     │
- *    │Average with Event Count│   │Average with Event Count│
- *    └───────────┬────────────┘   └─────────────┬──────────┘
- *                │                              │
- *                v                              v
- *  ┌───────────────────────────┐ ┌─────────────────────────────┐
- *  │  Write results to IMap    │ │  Write results to IMap      │
- *  └───────────────────────────┘ └─────────────────────────────┘
+ * <p>
```
Comment (eminn) [inline cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java]: Please do not format the diagram, can you revert this ?

File: cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java [hunk 2]
```diff
@@ -106,10 +112,20 @@ private static Pipeline buildPipeline() {
...
+                .readFrom(TwitterSources.timestampedStream(
+                        loadCredentials(),
+                        () -> new StatusesFilterEndpoint()
+                                .trackTerms(allCoinMarkers)))
                 .withNativeTimestamps(SECONDS.toMillis(1));
-
         StreamStageWithKey<Entry<CoinType, Double>, CoinType> tweetsWithSentiment = tweets
+                .map(rawTweet -> {
+                    try {
+                        return new JSONObject(rawTweet).getString("text");
```
Comment (eminn) [inline cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java]: Could it possible that Twitter returns a response without the `text` field ?

File: cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java (hunk 1 above)
Comment (ufukyilmaz) [reply cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java]: Sorry for this. I couldn't notice that the formatting broke the diagram. I've fixed it.

File: cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java (hunk 2 above)
Comment (ufukyilmaz) [reply cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java]: It normally returns the response with the `text` field. But in abnormal situations, I do not want to stop the job. The old version of this demo also had this kind of verification in the deleted `TwitterSource` file. In order to increase readability, I can remove this try-catch clause if you want.

Comment (eminn) [reply cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java]: This is for demonstration purposes only so I favor readability to please remove that

**Pull request merged**